### PR TITLE
Pocket Glocks

### PR DESCRIPTION
**Pull request by brndd**
[balance]

https://en.wikipedia.org/wiki/WP:BOLD

:cl:
 * tweak: NT Glocks fit in your pocket now. Subcompact tacticool EDC!

### CODE REVIEW

**Comment by Irradiation** [comment]
He did it, the absolute madman. That said, something something balance of having a "standard" gun taking fitting in something less large than a backpack/holster/belt.

**Comment by brndd** [comment]
Seriously speaking, it was said that the Glocks are generally kinda garbage compared to the other weapons available to sec. Perhaps this is the buff they need?

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 1]
```diff
@@ -160,11 +161,14 @@
...
 
 /obj/item/weapon/gun/projectile/glock/fancy
-	desc = "The NT Glock is a cheap, ubiquitous sidearm, produced by a NanoTrasen subsidiary. Uses .380AUTO rounds.<br><span class='notice'>This one has a sweet platinum-plated slide, and tritium night sights for maint crawling!</span>"
 	name = "\improper NT Glock Custom"
 	icon_state = "secgunfancy"
 	clowned = UNCLOWN
 
+/obj/item/weapon/gun/projectile/glock/fancy/New()
+	. = ..()
+	desc += "<br><span class='notice'>This one has a sweet platinum-plated slide, and tritium night sights for maint crawling!</span>"
```
Comment (boy2mantwicethefam) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Should replace maint with maintenance, you know, just for the looks

Comment (brndd) [reply code/modules/projectiles/guns/projectile/pistol.dm]: it is done

**Comment by ghost** [comment]
Pocket slots are pretty valuable so it is still not an easy choice for the average powergame sec.

**Comment by brndd** [comment]
The real reason I think this will be cool is that you can print infinite Glocks from the ammolathe and give them to the crew as convenient personal defense weapons, and now they can actually conveniently carry them around.

**Comment by jknpj** [comment]
I'd personally add them to every loadout :us:

**Comment by Blithering** [comment]
I'd prefer if there were both the regular and subcompact glock type with a smaller clip size for the pocket version, so there's the choice between storage or capacity.
Glock-19-vs-26

**Comment by brndd** [comment]
The current glocks *are* subcompact, that's the point of this PR.

**Comment by Blithering** [comment]
Guess I was thinking more on the lines of a Glock 43, with the smaller capacity

**Pull request merged**